test/ctmrg/suweight.jl

```diff
@@ -13,27 +24,35 @@ function rand_wts(peps::InfinitePEPS)
     return wts
 end
 
-function su_rdm_1x1(row::Int, col::Int, peps::InfinitePEPS, wts::SUWeight)
+function su_rdm_1x1(
+        row::Int, col::Int, peps::InfinitePEPS, wts::Union{Nothing, SUWeight} = nothing
+    )
     Nr, Nc = size(peps)
     @assert 1 <= row <= Nr && 1 <= col <= Nc
     t = peps.A[row, col]
-    t = absorb_weight(t, wts, row, col, Tuple(1:4))
-    ρ = t * t'
+    if !(wts === nothing)
+        t = absorb_weight(t, wts, row, col, Tuple(1:4))
+    end
+    # contract local ⟨t|t⟩ without virtual twists
+    @tensor ρ[k; b] := conj(t[b; n e s w]) * twistdual(t, 2:5)[k; n e s w]
```

Comment:
```suggestion
    # contract local ⟨t|t⟩ without virtual twists
    @plansor ρ[k; b] := conj(t[b; n e s w]) * t[k; n e s w]
```
But this is also just `t' * t`